aws/data_source_aws_wafregional_ipset_test.go

```diff
@@ -0,0 +1,51 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
```

Comment:
Nit: Imports should be sorted and alphabetical into stdlib - empty line - third party. Tools like `goimports` can help do this for you automatically. We'd turn on the `golangci-lint` testing for this, but its very slow given this project's size.